### PR TITLE
Add notice docs

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

Add a notice for the fraud proof upgrades

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Add a new section and add docs

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

File: .gitbook/contents/notices/fraud-proofs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Prepare for Fraud Proofs on Boba Sepolia and Boba BNB Testnet 
+
+Fraud proofs are important for making L2 more decentrailized, allowing more community involvement in validating the L2 state. These changes allow anyone to withdraw funds from Boba Layer 2 to L1without relying on any centralized actors.
```
Comment (jyellick) [inline .gitbook/contents/notices/fraud-proofs.md]: `L1without` is run together.

Perhaps though something like

These changes allow anyone to submit or challenge a state root, an essential part of the process used for withdrawing funds from L2 to L1.

File: .gitbook/contents/notices/fraud-proofs.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Prepare for Fault Proofs
+
+Fault proofs are expected to go live for Boba Testnets (Boba Sepolia and Boba BNB Testnet) in mid-August.
```
Comment (jyellick) [inline .gitbook/contents/notices/fraud-proofs.md]: Add a year maybe, e.g. mid-August 2024?

File: .gitbook/contents/notices/fraud-proofs.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Changes for Testnet Withdrawals
+
+* Withdrawal will require proving and finalizing based on the fault proof system.
+
+* Withdrawal will require 7 days to finalize depending on the outcome of the dispute game. 
+
+  > Boba Testnets withdrawals are no longer instant.
+
+* The `DisputeGameFactory` will replace the `L2OutputOracle` contract for prosing output root statements, enhancing platform security and reliability.
```
Comment (jyellick) [inline .gitbook/contents/notices/fraud-proofs.md]: s/prosing/proposing/

File: .gitbook/contents/notices/fraud-proofs.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+* The valid proofs challenged by maliciously players can be delayed up to an anditional a few hours at a very high cost to the malicious actor.
```
Comment (jyellick) [inline .gitbook/contents/notices/fraud-proofs.md]: s/The/Any/

s/can be/can only be/

s/anditional a/additional/

File: .gitbook/contents/notices/fraud-proofs.md (hunk 1 above)
Comment (boyuan-chen) [reply .gitbook/contents/notices/fraud-proofs.md]: Fixed

File: .gitbook/contents/notices/fraud-proofs.md (hunk 2 above)
Comment (boyuan-chen) [reply .gitbook/contents/notices/fraud-proofs.md]: Fixed

File: .gitbook/contents/notices/fraud-proofs.md (hunk 4 above)
Comment (boyuan-chen) [reply .gitbook/contents/notices/fraud-proofs.md]: Fixed

File: .gitbook/contents/notices/fraud-proofs.md (hunk 3 above)
Comment (boyuan-chen) [reply .gitbook/contents/notices/fraud-proofs.md]: Fixed

**Review by mmontour1306 — APPROVED** [review]

**Review by jyellick — APPROVED** [review]

**Pull request merged**